io-engine/src/bdev/nexus/nexus_child.rs

```diff
@@ -116,8 +116,6 @@ pub enum Reason {
     /// No particular reason for the child to be in this state.
     /// This is typically the init state.
     Unknown,
-    /// Out of sync: child device is ok, but needs to be rebuilt.
-    OutOfSync,
```

Comment:
Guess we need to update protobuf as well?

Reply:
No, not at this moment. I just remapped new child flags into old protobuf states.

Reply:
Ok but if we don't use it might be worth removing otherwise could be misleading: https://github.com/openebs/mayastor-api/blob/6487977a5785510b9c5864bed561cd2c09170b79/protobuf/v1/nexus.proto#L96

Reply:
I've added a ticket to cover this and other things: https://dcsw-bolt.atlassian.net/browse/B2-842